lib/util/time.ex

```diff
@@ -37,8 +37,20 @@ defmodule Antikythera.Time do
   Convert timestamps into `Antikythera.Time.t`, leveraging `recursive_new?` option of `Croma.Struct`.
 
   Only `Antikythera.IsoTimestamp.t` can be converted.
+
+      iex> {:ok, time} = #{__MODULE__}.new("2015-01-23T23:50:07Z")
+      {:ok, {#{__MODULE__}, {2015, 1, 23}, {23, 50, 7}, 0}}
+      iex> #{__MODULE__}.new(time)
+      {:ok, {#{__MODULE__}, {2015, 1, 23}, {23, 50, 7}, 0}}
+      iex> #{__MODULE__}.new("2015-01-23T23:50:07") |> Croma.Result.error?()
+      true
+      iex> #{__MODULE__}.new(nil) |> Croma.Result.error?()
+      true
   """
-  defun new(s :: v[IsoTimestamp.t()]) :: R.t(t), do: from_iso_timestamp(s)
+  defun new(t | IsoTimestamp.t()) :: R.t(t) do
```

Comment:
The document of this function says:
```
  Convert timestamps into `Antikythera.Time.t`, leveraging `recursive_new?` option of `Croma.Struct`.
```

but, `new/1` accepts `Antikythera.Time.t` now.

Could you fix the document?

Reply:
Thank you!
I should update the document.

fixed at https://github.com/access-company/antikythera/pull/204/commits/32fb7f59908af016a3badb59e9c4c6a892466136